Formula/bacula-fd.rb

```diff
@@ -42,13 +42,7 @@ def install
     system "make", "install"
 
     # Avoid references to the Homebrew shims directory
-    if OS.mac?
-      inreplace Dir[prefix/"etc/bacula_config"],
-                HOMEBREW_SHIMS_PATH/"mac/super/", ""
-    else
-      inreplace Dir[prefix/"etc/bacula_config"],
-                HOMEBREW_SHIMS_PATH/"linux/super/", ""
-    end
+    inreplace Dir[prefix/"etc/bacula_config"], Superenv.shims_path, ""
```

Comment:
```suggestion
    inreplace Dir[prefix/"etc/bacula_config"], "#{Superenv.shims_path}/", ""
```
We need the trailing `/` to avoid turning what should have been `clang` into `/clang`, unfortunately.

Alternatively:
```suggestion
    inreplace Dir[prefix/"etc/bacula_config"] do |s|
      s.gsub! Superenv.shims_path/ENV.cc, ENV.cc
      s.gsub! Superenv.shims_path/ENV.cxx, ENV.cxx
    end
```